File: Class_member/3.rb
class Cs
  @@count = 0
  def initialize()
    @@count = @@count + 1
    # not instance, it connected class
  end
  def Cs.getCount()
    return @@count
  end
end

i1 = Cs.new()
i2 = Cs.new()
i3 = Cs.new()
p Cs.getCount()
# class variable = all class co-use

File: Class_member/3.py
class Cs:
    count = 0
# class inside + method outside variable = class variable
    def __init__(self):
        # method inside connection with class variable
        Cs.count = Cs.count +1
    @classmethod
    def getCount(cls):
        # cls = this method-class var
        return Cs.count
        # == cls.count

i1 = Cs()
i2 = Cs()
i3 = Cs()
print(Cs.getCount())
